client/components/editor/Sidebar/Discussion.vue

```diff
@@ -3,17 +3,39 @@
     <discussion
       @change="$emit('change', $event)"
       :activity="activity"
+      :is-visible="isVisible"
       show-notifications />
   </div>
 </template>
 
 <script>
+import { mapGetters, mapMutations } from 'vuex';
 import Discussion from '@/components/repository/common/Sidebar/Discussion/Embed';
+import get from 'lodash/get';
 
 export default {
   name: 'editor-discussion',
   props: {
-    activity: { type: Object, required: true }
+    activity: { type: Object, required: true },
+    isVisible: { type: Boolean, default: false }
+  },
+  computed: {
+    ...mapGetters('repository/comments', ['getActivityComments']),
+    comments: vm => vm.getActivityComments(vm.activity.id),
```

Comment:
You can just listen to the change event.

Reply:
Not sure how that can help since child component emits all comments (not filtered by activityId)

Reply:
That's an error then ;) It should only report ones related to current activity. 
If you decide to keep it like this, then remove that event since it is no longer used 😉